src/seedu/addressbook/data/person/Address.java

```diff
@@ -36,6 +39,11 @@ public static boolean isValidAddress(String test) {
         return test.matches(ADDRESS_VALIDATION_REGEX);
     }
 
+    /**
+     * Retrieves a listing of every word in the address, in order.
+     */
+    public List<String> getWordsInAddress() { return Arrays.asList(value.split("\\s+")); }
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. brace style problem.